.pylintrc

```diff
@@ -13,7 +13,7 @@ ignore=third_party
 
 # Add files or directories matching the regex patterns to the blacklist. The
 # regex matches against base names, not paths.
-ignore-patterns=prediction_pb2.py,prediction_pb2_grpc.py,mnist_DDP.py
+ignore-patterns=prediction_pb2.py,prediction_pb2_grpc.py,mnist_DDP.py,mnistddpserving.py
```

Comment:
Why ignore mnistddpserving.py? Is this code copied from somewhere else?

Reply:
Thanks - Not really, I was getting errors with the method forward() as pylint expected an arbitrary argument list in the method signature def forward(self, *x), I will try to fix the method signature to make it work in both the trainers and serving modules in the next release (Original examples from where I took code snippets have same problem).
Pylint error "Parameters differ from overridden 'forward' method (37:2) [arguments-differ]"